detection_rules/version_lock.py

```diff
@@ -240,6 +240,12 @@ def log_changes(r, route_taken, new_rule_version, *msg):
                 elif min_stack > latest_locked_stack_version:
                     route = 'B'
                     # 3) on the latest stack, locking in a breaking change
+
+                    # preserve buffer space to support forked version spacing
+                    if exclude_version_update:
+                        buffer_int -= 1
```

Comment:
This is to make sure we dont also include the rule version autobump

Reply:
if excluding version update, shouldn't the buffer be skipped too?